apps/emqx_authz/src/emqx_authz_api_sources.erl

```diff
@@ -445,7 +445,8 @@ read_certs(#{<<"ssl">> := SSL} = Source) ->
     end;
 read_certs(Source) -> Source.
 
-maybe_write_certs(#{<<"ssl">> := #{<<"enable">> := true} = SSL} = Source) ->
+maybe_write_certs(#{<<"ssl">> := #{<<"enable">> := True} = SSL} = Source)
+  when (True =:= true) or (True =:= <<"true">>) ->
```

Comment:
Nit: IS_TRUE

Reply:
🆗 Will update this in one of the following authz improvements.